pages/learner-onboarding/avatar-selection.js

```diff
@@ -2,11 +2,17 @@ import LearnerOnboardingLayout from "../../components/LearnerOnboardingLayout";
 import styled from "styled-components";
 import Image from "next/image";
 import avatars from "../../data/avatars.js";
+import { useEffect } from "react";
 
 export default function LearnerAvatarSelection({
+  background,
   avatarNameInLocalStorage,
   setAvatarNameInLocalStorage,
 }) {
+  useEffect(() => {
+    document.body.style.backgroundColor = background;
+  }, []);

```

Comment:
Rather than put this in every ``learner-onboarding/`` page, couldn't it just go in the ``LearnerOnboardingLayout.js`` component?